rocketmq-connect/rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/rest/RestSender.java

```diff
@@ -0,0 +1,38 @@
+package org.apache.rocketmq.connect.runtime.rest;
```

Comment:
We need to add Apache header at the start of each file~

Reply:
I have added Apache header at the start of each file